src/_base/harness/config/functions.yml

```diff
@@ -253,8 +253,14 @@ function('replace', [haystack, needle, replacement]): |
 
 function('template_key_value', [template, key_value]): |
   #!php
+  if ($key_value === null) {
+    return null;
```

Comment:
not sure if empty array is more appropriate

Reply:
Good idea. I will change it to this:
```php
if (empty($key_value)) {
    return [];
}
```